travelpal/lib/travelpal_web/controllers/flight_controller.ex

```diff
@@ -1,6 +1,14 @@
 defmodule TravelpalWeb.FlightController do
```

Comment:
Looks like this module is missing some functions that are currently in `master` like `show()` and `update()`. You could look at the [current flight controller on master](https://github.com/kimberlypn/TravelPal/blob/master/travelpal/lib/travelpal_web/controllers/flight_controller.ex) and see if you have made any changes since. If not, I would just copy and paste the file.